src/main/java/in/projecteka/consentmanager/hipLinkInitialization/UserAuthService.java

```diff
@@ -0,0 +1,18 @@
+package in.projecteka.consentmanager.hipLinkInitialization;
+
+import in.projecteka.consentmanager.hipLinkInitialization.model.request.AuthInitRequest;
+import lombok.AllArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Mono;
+
+@AllArgsConstructor
+public class UserAuthService implements HIPLinkInitializer {
```

Comment:
I would have thought that the UserAuthService will always be there, but specific implementation of HIPLInkInit may not be there. So UserAuthService should be composed of instances of HIPLinkInitializer

Reply:
done